src/Event/AfterSendEvent.php

```diff
@@ -22,13 +23,20 @@ class AfterSendEvent extends Event
     private $response;
 
     /**
+     * @var RequestInterface
+     */
+    private $request;
+
+    /**
      * Constructor
      *
      * @param ResponseInterface $response
+     * @param RequestInterface  $request
      */
-    public function __construct(ResponseInterface $response = null)
+    public function __construct(ResponseInterface $response = null, RequestInterface $request = null)
```

Comment:
Would it be better to switch the order of these arguments? And would they ever be `null`?

```php
public function __construct(RequestInterface $request, ResponseInterface $response)
```